ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1585,7 +1738,7 @@ class RangeSelector {
 
             let xOffsetForExportButton = 0;
 
-            if (inputGroup) {
+            if (options.inputEnabled && inputGroup) {
```

Comment:
Q: Did something change I missed? `input.group` shouldn't exist if inputs are disabled, right?

Reply:
The `alignElements` method is fired multiple times. On the update it is fired 2 times. If you update the input to be disabled, in the first iteration the `options.inputEnabled`option if false, but `inputGroup`still exists. It gets deleted later. So I did it in sake performance.